src/bin/server.ts

```diff
@@ -46,16 +46,67 @@ interface Args {
   if (configPath) {
     try {
       options = require(configPath);
-      // make sure that networkKey is passed as buffer.
-      // accept both zwave2mqtt format as ozw format
-      if (options.networkKey && options.networkKey.length === 32) {
-        options.networkKey = Buffer.from(options.networkKey, "hex");
+      const securityKeyNames = [
+        "S0_Legacy",
+        "S2_AccessControl",
+        "S2_Authenticated",
+        "S2_Unauthenticated",
+      ];
+      // We prefer the securityKeys option over the networkKey one
+      for (const key of securityKeyNames) {
+        if (
+          options.securityKeys &&
+          options.securityKeys[key] &&
+          options.securityKeys[key].length === 32
+        ) {
+          options.securityKeys[key] = Buffer.from(
+            options.securityKeys[key],
+            "hex"
+          );
+        } else if (
+          options.securityKeys &&
+          options.securityKeys[key] &&
+          options.securityKeys[key].includes("0x")
+        ) {
+          options.securityKeys[key] = options.securityKeys[key]
+            .replace(/0x/g, "")
+            .replace(/, /g, "");
+          options.securityKeys[key] = Buffer.from(
+            options.securityKeys[key],
+            "hex"
+          );
+        } else if (options.securityKeys && options.securityKeys[key]) {
+          console.error(
+            "Error: Invalid securityKeys." + key + " value defined."
+          );
+          return;
+        }
+      }
```

Comment:
node-zwave-js does not require it, but should the server require that all the possible keys are defined? It does (or did?) for `networkKey`.

Reply:
I think we can get more strict about this later, but we want to also make sure that users can upgrade cleanly without breaking things for them.